### DISCUSSION TITLE
Is it possible to detect clapping?

### DISCUSSION THREAD

**Discussion by Thresher12** [openai/whisper#1489] [Q&A]
Is it possible to use whisper to detect clapping? Even when mixed with talking? I need to scan audio files. Most of the other solutions I see are designed to detect real time input from microphones and I'm not sure how well they'd do with mixed clapping and speech. 

Also I'd like to detect laughter as well if possible but clapping by itself is good since I figure laughter will be harder.

**Comment by phineas-pta** [comment]
this is speech recognition not sound classification, try other models

**Comment by phineas-pta** [comment]
update: u can try https://github.com/YuanGongND/whisper-at
